extensions/mssql/src/reactviews/common/errorDialog.tsx

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogBody,
+    DialogContent,
+    DialogSurface,
+    DialogTitle,
+    makeStyles,
+    shorthands,
+} from "@fluentui/react-components";
+import { ErrorCircleRegular } from "@fluentui/react-icons";
+import React from "react";
+
+const useStyles = makeStyles({
+    dialogSurface: {
+        minWidth: "320px",
+        maxWidth: "420px",
+    },
+    title: {
+        display: "flex",
+        alignItems: "center",
+        columnGap: "8px",
+    },
+    icon: {
+        fontSize: "32px",
+    },
+    content: {
+        ...shorthands.marginBlock("16px", "0"),
+    },
+});
+
+export interface ErrorDialogProps {
+    open: boolean;
+    title: string;
+    message: string;
+    retryLabel: string;
+    onRetry: () => void;
+}
+
+export const ErrorDialog: React.FC<ErrorDialogProps> = ({
+    open,
+    title,
+    message,
+    retryLabel,
+    onRetry,
+}) => {
+    const classes = useStyles();
+    return (
+        <Dialog open={open} modalType="modal" inertTrapFocus>
```

Comment:
Fine for this PR, but in the future, please make sure that PRs with UX changes include screenshots!

Reply:
Added.